apps/course-authoring/src/renderer/pages/home/index.tsx

```diff
@@ -92,7 +92,7 @@ export const Element = () => {
 };
 
 export default {
-  Name,
-  Route,
+  PageName,
+  PageRoute,
   Element,
```

Comment:
If you are going to change the naming convention of name and route, shouldn't element being changed as well?

So that it is `PageElement`.

Or maybe even

```js
export default {
    Page: {
        Name,
        Route,
        Element
    }
}
```

Reply:
Done. I ended up using `PageElement`.

The reason I changed this naming convention was that I had to use react Route and the page Route in the same component, I could have renamed one of them using an alias but it was better to rename it and avoid that issue in the future.